packages/react/src/components/Tile/Tile.js

```diff
@@ -196,10 +196,8 @@ export class ClickableTile extends Component {
         href={href}
         className={classes}
         {...rest}
-        // TODO: replace with onClick when handleClick prop is deprecated
```

Comment:
I actually didn't mind these comments, @joshblack do you have a preference for tracking things we'll need to rip out post-v11.